docs/4.0/docs/build-cloud-image/build-cloudImage-with-helm-charts.md

```diff
@@ -152,4 +153,80 @@ If you use a private image repository, just login registry with `sealos login` b
 sealos login docker.io -u xxx -p xxx
 
 sealos login registry.cn-hangzhou.aliyuncs.com -u xxx -p xxx
-```
\ No newline at end of file
+```
+
+## Install cloud image
+
+Then you can run cloud image in your cluster.
+
+```shell
+sealos run labring/nginx:v1.23.2
+```
+
+The helm binary command will installed to your master nodes of your kubernetes cluster.
+
+```shell
+root@ubuntu:~# helm -n nginx ls
+```
+
+## Custom values.yaml
+
+By default, when building an image, sealos only parses the default values.yml, However, you can also provide a custom values.yaml for sealos.
+
+The custom values.yaml must same layer with your chart , and file name must be `<chart-name>.values.yaml`,eg`loki-stack.values.yaml`
+
+```shell
+.
+├── charts
+│   ├── loki-stack
+│   │   ├── charts
+│   │   ├── Chart.yaml
+│   │   ├── README.md
+│   │   ├── requirements.lock
+│   │   ├── requirements.yaml
+│   │   ├── templates
+│   │   └── values.yaml
+│   └── loki-stack.values.yaml
+├── init.sh
+├── Kubefile
+```
+
+The `loki-stack.values.yaml` like this:
+
+```shell
+$ cat charts/loki-stack.values.yaml
+promtail:
+  enabled: false
+fluent-bit:
+  enabled: true
+grafana:
+  enabled: true
+```
+
+Different values.yaml files will parses different images, which is the purpose of customizing values.yml - to enable sealos to automatically parses the images during run `sealos build` .
```

Comment:
Different values files may be output to a different list of images

to enable sealos to automatically parses the images during `sealos build`.